formatTest/typeCheckedTests/expected_output/jsx.re

```diff
@@ -95,8 +95,8 @@ module Namespace = {
 };
 
 module Optional1 = {
-  let createElement ::optional children =>
-    switch optional {
+  let createElement ::requiredOpt children =>
```

Comment:
Yeah, that was the idea behind the `requiredOpt` naming. I'll rename it to `required` to avoid any confusion.